lms/djangoapps/instructor/views/api.py

```diff
@@ -659,8 +659,8 @@ def create_and_register_users_without_email(request):
     for email in list(new_users_to_register):
         identifier = email.split('@')[0]
         details = identifiers_raw.get(identifier)
-        name = '{}'.format(details.get('firstname'))
-        last_name = '{}'.format(details.get('lastname'))
+        name = details.get('firstname')
+        last_name = details.get('lastname')
```

Comment:
These changes don't seem to be related to the Report, please remove from this PR. I think they are linked to another change

Reply:
this PR contains 2 resolved issues and 1 feature. this is part of "fix for UnicodeDecode error for instructor API" code

Reply:
@nerfani UAT instance is intended to test the current state of application. In case you want to deploy a part of currently deployed on UAT changes we need to deploy only these changes on UAT, test them and then deploy on production.
This in not a feature PR - it is a sync PR from UAT to production.